HelpHunterBE/Controllers/OrderController.cs

```diff
@@ -0,0 +1,32 @@
+
+using HelpHunterBE.Logic;
+using HelpHunterBE.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+
+namespace HelpHunterBE.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    [AllowAnonymous]
```

Comment:
Na pewno chcemy zamknąć składanie zamówień dla zalogowanych użytkowników
```suggestion
    [Authorize]
```